src/products/components/ProductVariantPage/ProductVariantPage.tsx

```diff
@@ -313,6 +336,9 @@ const ProductVariantPage: React.FC<ProductVariantPageProps> = ({
                     warehouses={warehouses}
                     onChange={handlers.changeStock}
                     onFormDataChange={change}
+                    onEndPreorderTrigger={
+                      variant?.preorder.isPreorder ? togglePreorderModal : null
```

Comment:
```
variant?.preorder.isPreorder
```
Isn't this already checked inside `ProductStocks` component?

Reply:
Yes it is - fixed